src/daos/metadata_dao.py

```diff
@@ -0,0 +1,115 @@
+import csv
+import os
+from datetime import datetime
+from typing import Dict, Any, List
+import logging
+
+logger = logging.getLogger(__name__)
+
+FIELD_NAMES = ['market_slug', 'market_id', 'asset_id', 'outcome_name', 'run_at', 'game_start_at', 'game_end_at']
```

Comment:
```suggestion
FIELD_NAMES = ['market_slug', 'market_id', 'asset_id', 'outcome_name', 'executed_timestamp', 'game_start_timestamp']
```

1. We don't need `game_end_at`. You can remove from everywhere in the branch as well.
2. I changed my mind and want to rename `run_at` to `executed_at`. Please update everywhere else and update tests
3. I also want to rename `game_start_at` to `game_start_timestamp`. Please update everywhere else and update tests